README.md

```diff
@@ -50,6 +60,27 @@ npm install
 cd ../..
 ```
 
+# Setting Up Vision System (Windows)
+See **Vision System Troubleshooting** section of this [troubleshooting guide](https://docs.google.com/document/d/17iD53BYdfiRP9ht-XtAxGYFpu55B4e1CCysB0ldnopU/edit?usp=sharing) for visual steps
+1. Open **cs-reminibot** directory in File Explorer.
+2. Navigate to the **installation_scripts** directory.
+3. Right click on the **Vision_system_install** file.
+4. Select the **Open with** option from the resulting drop down.
+5. Select the **Git for Windows** option from the resulting sub-drop down
+
+# Setting Up Vision System (MacOS/Linux)
+Run the following commands to navigate to the apriltag-py directory in vision/apriltag-py and install JavaScript dependencies.
+On Linux or MacOS open terminal and run the commands. You should currently be in the cs-reminibot directory
+```
+cd vision/apriltag-py
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release
+make -j4
```

Comment:
I was running into error: unknown target CPU 'vortex'
seems like it might be an issue with M1. please include somewhere in the README that it's a known issue with M1